doc/content/examples/example11b.md

```diff
@@ -0,0 +1,94 @@
+# Example 11b: Mesh refinement using `Marker` and `Indicator`, and demonstration of `SidesetMoment` and `AverageValue` postprocessors
+
+This example expands on [Example 11a](examples/example11.md) to demonstrate mesh refinement using `Markers` and `Indicators`. This example also demonstrates the use and verification of `SidesetMoment` and `AverageValue` postprocessors. For demonstration purposes,
+the beam model subjected to static loading from [Example 11a](examples/example11.md) is borrowed. This beam model is divided into three subdomains represented using three colors (red, green, and gray) in the figure below.
+
+!media media/examples/ex11b/ex11b_blocks.png
+       style=width:50%;margin-left:150px;float:center;
+       id=fig:beam_blocks
+       caption=Beam model with the domain divided into three subdomains. The red, green, and gray colors represent three subdomains.
```

Comment:
Might want to specify if these are three different blocks instead of just calling them subdomains.

Reply:
Instead of blocks, I preferred subdomains. I also made sure to use subdomains throughout.